lib/avalara.rb

```diff
@@ -147,8 +147,10 @@ def self.authentication
 
   def self.address_match? address, response_address
     state = ((address.state && address.state.abbr) || (address.state_name || ''))
-    response_address["City"].eql?(address.city) && response_address["Region"].eql?(state) &&
-      response_address["PostalCode"].split('-').first.eql?(address.zipcode) && response_address["Country"].eql?(address.country.iso)
+    response_address["City"].downcase.eql?(address.city.downcase) &&
```

Comment:
why not downcase everything before with something like 
`hash.each {|k,v| v.downcase!}`